docs/edgeql/sdl/links.rst

```diff
@@ -47,10 +47,42 @@ commands <ref_eql_ddl_links>`.
       [ extending <base> [, ...] ] -> <type>
       [ "{" <subcommand>; [...] "}" ] ;
 
+      [ "{"
+          [ on target delete <action> ]
+          [ <property-declarations> ]
+          [ <constraint-declarations> ]
+          [ <attribute-declarations> ]
+          ...
+        "}" ]
+
+
     # Computable link form used inside type declaration:
     [ required ] [{single | multi}] link <name> := <expression>;
 
     # Abstract link form:
-    abstract link [<module>::]<name> [extending <base> [, ...]]
-    [ "{" <subcommand>; [...] "}" ]
+    abstract link <name> [extending <base> [, ...]]
+    [ "{"
+        [ <property-declarations> ]
+        [ <constraint-declarations> ]
+        [ <index-declarations> ]
+        [ <attribute-declarations> ]
+        ...
+      "}" ]
+
+Description
+-----------
+
+Much of the syntax is the same as for :ref:`DDL <ref_eql_ddl_links>`.
+The main difference is in the nested declarations:
```

Comment:
I guess we better add an example in places like this one. "Much of the syntax is the same" doesn't cut it.

Reply:
The example *and* the synopsis already appear earlier in that page. The description of what the synopsis means is long and identical to DDL, hence the link.